azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/ConfigMojo.java

```diff
@@ -0,0 +1,340 @@
+        }
+
+        if (targetProjects.size() == 0) {
+            getLog().warn("There are no projects in current folder which are not configured yet!");
+            return;
+        }
+        selectProjects();
+    }
+
+    private void selectProjects() throws MojoFailureException, IOException {
+        targetProjects = prompt.promoteMultipleEntities("project", targetProjects, MavenProject::getName, false, " for ALL projects", targetProjects);
+    }
+
+    private void configureJvmOptions() throws IOException {
+        this.deployment.withJvmOptions(prompt.promoteString("jvmOptions", this.deployment.getJvmOptions(), ".*", false));
+    }
+
+    private void configureCpu() throws IOException {
+        // TODO: default value
+        this.deployment.withCpu(prompt.promoteInteger("cpu", 1, 1, 64, true));
+    }
+
+    private void configureMemory() throws IOException {
+        // TODO: default value
+        this.deployment.withMemoryInGB(prompt.promoteInteger("memory(GB)", 2, 1, 128, true));
+    }
+
+    private void configureInstanceCount() throws IOException {
+        // TODO: default value
+        this.deployment.withInstanceCount(prompt.promoteInteger("instanceCount", 1, 1, 10, true));
+    }
+
+    private void confirmAndSave() throws IOException {
+        System.out.println("Please confirm the following properties");
+        System.out.println("subscription id: " + TextUtils.green(this.subscriptionId));
+        System.out.println("service name   : " + TextUtils.green(this.clusterName));
+        System.out.println("app name       : " + TextUtils.green(this.appName));
+        if (this.isPublic != null && this.isPublic.booleanValue()) {
+            System.out.println("public access  : " + TextUtils.green(this.isPublic.toString()));
+        }
+        if (this.deployment.getInstanceCount() != null) {
+            System.out.println("instance count : " + TextUtils.green(this.deployment.getInstanceCount().toString()));
+        }
+
+        if (this.deployment.getCpu() != null) {
+            System.out.println("cpu            : " + TextUtils.green(this.deployment.getCpu().toString()));
+        }
+
+        if (this.deployment.getMemoryInGB() != null) {
+            System.out.println("memory(GB)     : " + TextUtils.green(this.deployment.getMemoryInGB().toString()));
+        }
+
+        if (StringUtils.isNotEmpty(this.deployment.getJvmOptions())) {
+            System.out.println("jvm options    : " + TextUtils.green(this.deployment.getJvmOptions()));
+        }
+
+        if (prompt.promoteYesNo(true, "Confirm to save all the properties listed above?", true)) {
+            saveConfigurationToProject(this.project, true);
+            getLog().info("These changes are saved to the pom.xml file.");
+        }
+    }
+
+    private void confirmAndSaveParent() throws IOException {
+        System.out.println("Please confirm the following properties");
```

Comment:
use `String.format("%-20s %s",attribute,value)` instead of space?

Reply:
done